modules/module-extras.php

```diff
@@ -82,3 +82,25 @@ function jetpack_widgets_add_suffix( $widget_name ) {
 	);
 }
 add_filter( 'jetpack_widget_name', 'jetpack_widgets_add_suffix' );
+
+add_action( 'blog_privacy_selector', 'priv_notice_privacy_selector' );
+
+/**
```

Comment:
It may be worth detailing that comment a bit more, since it's unusual to have such settings in this file.

Reply:
Implemented in [6326e10](https://github.com/Automattic/jetpack/pull/10333/commits/6326e102e91c2fe90413a43b13d16a04e3a65cf5)

Reply:
I'll mark this as resolved, but the sha link above isn't pointing to anything anymore.